sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -24,14 +24,13 @@ export interface ProgramOfferingBaseDTO {
   offeringWILType?: string;
   studyBreaks?: StudyBreak[];
   offeringDeclaration: boolean;
+  assessedBy?: string;
+  assessedDate?: Date;
+  submittedDate?: Date;
```

Comment:
The `submittedDate` will always be present, right?

Reply:
Currently create and update API are using the same DTO. So kept submittedDate as optional as it is only for create scenario.